.github/workflows/12-backend-jacoco.yml

```diff
@@ -0,0 +1,74 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: "12-backend-jacoco: Java Test Coverage (Jacoco)"
+
+on:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        required: true
+      TEST_PROPERTIES:
+        required: false
```

Comment:
```suggestion
```

Let's remove this.   It turns out that TEST_PROPERTIES is left over from functionality that we no longer use or need.